### ISSUE TITLE
ascend-test2's disk is filling up 

### ISSUE THREAD

**Issue by ian-noaa** [NOAA-GSL/VxIngest#414]
## Describe the bug
The ascend-test2 VM is filling the following disk partitions. We need to figure out why.

* `/`
* `/var`
* `/data-ingest`

## To Reproduce
TBD

1. Create a GRIB filepath that exists when `builder_common/vx_ingest.py/get_file_list()` is called
2. Remove the file after the file list is made
3. Run `grib_buider_parent.py/build_document()` and see a large number of stack traces

## Expected behavior
VxIngest shouldn't be filling up VM disks.

**Comment by ian-noaa** [comment]
Turns out the following were filling up:

* `/` - the amb-verif user has large logs
* `/data-ingest` - the large logs were copied over here
* `/var` - the container was storing a copy of the large logs here

The root of these abnormally large logs is that files were missing when we went to process them. This was causing tracebacks in the log output. We have two issues:

1. Our code isn't properly catching when we fail to open a file. We should log a message with the failed file path and move on and suppress the stacktrace from cfgrib/xarray.
2. For some reason files are disappearing between when we build the file list and when we attempt to process the file.
